dev-libs/libtimezonemap/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
```

Comment:
I'm not exactly clear on the whole proxy-maint process, since I haven't found any detailed docs and my questions in the bug weren't answered.

I assumed that since Cinnamon is the only consumer (and likely to be the only consumer) that it would be owned by the Cinnamon project (and Cinnamon proxy maintainers would have been implied). Or should I be adding myself as proxy-maint on all of these packages?

Reply:
There is no cinnamon proxy maintainers.  
You should add yourself and proxy-maintainers to all packages.

Thanks a lot for doing this BTW, its a huge thing to pull off.